src/components/Votes.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react";
+import { patchVotesByArtcleId } from "../utils/api"
+
+const Votes = ({ votes, article_id }) => {
+    const [votesChange, setVotesChange] = useState(0);
+
+    function incVote() {
```

Comment:
We could look at using a single function for both up and down voting - currently this code is very DRY.